connection_test.go

```diff
@@ -37,13 +37,18 @@ func TestMConnectionSend(t *testing.T) {
 
 	msg := "Ant-Man"
 	assert.True(mconn.Send(0x01, msg))
-	assert.False(mconn.CanSend(0x01))
+	assert.False(mconn.CanSend(0x01), "CanSend should return false because queue is full")
```

Comment:
don't think this is deterministic because a started connection will be reading from the sendQueue. This is why the following assertions fail too.